CI `Check re-exports` fails on master

A recent change to master causes the CI check to fail:
<details> <summary>Click to expand error details</summary>
error[E0432]: unresolved import `bitcoin::amount::BadPositionError`
  --> bitcoin/tests/check-re-exports.rs:71:9
   |
71 |     use bitcoin::amount::BadPositionError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `BadPositionError` in `amount`

error[E0432]: unresolved import `bitcoin::amount::InputTooLargeError`
  --> bitcoin/tests/check-re-exports.rs:74:9
   |
74 |     use bitcoin::amount::InputTooLargeError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `InputTooLargeError` in `amount`

error[E0432]: unresolved import `bitcoin::amount::InvalidCharacterError`
  --> bitcoin/tests/check-re-exports.rs:75:9
   |
75 |     use bitcoin::amount::InvalidCharacterError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `InvalidCharacterError` in `amount`

error[E0432]: unresolved import `bitcoin::amount::MissingDenominationError`
  --> bitcoin/tests/check-re-exports.rs:76:9
   |
76 |     use bitcoin::amount::MissingDenominationError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `MissingDenominationError` in `amount`
   |
help: a similar name exists in the module
   |
76 -     use bitcoin::amount::MissingDenominationError as _;
76 +     use bitcoin::amount::ParseDenominationError as _;
   |

error[E0432]: unresolved import `bitcoin::amount::MissingDigitsError`
  --> bitcoin/tests/check-re-exports.rs:77:9
   |
77 |     use bitcoin::amount::MissingDigitsError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `MissingDigitsError` in `amount`

error[E0432]: unresolved import `bitcoin::amount::PossiblyConfusingDenominationError`
  --> bitcoin/tests/check-re-exports.rs:82:9
   |
82 |     use bitcoin::amount::PossiblyConfusingDenominationError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `PossiblyConfusingDenominationError` in `amount`

error[E0432]: unresolved import `bitcoin::amount::TooPreciseError`
  --> bitcoin/tests/check-re-exports.rs:84:9
   |
84 |     use bitcoin::amount::TooPreciseError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `TooPreciseError` in `amount`

error[E0432]: unresolved import `bitcoin::amount::UnknownDenominationError`
  --> bitcoin/tests/check-re-exports.rs:85:9
   |
85 |     use bitcoin::amount::UnknownDenominationError as _;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `UnknownDenominationError` in `amount`
   |
help: a similar name exists in the module
   |
85 -     use bitcoin::amount::UnknownDenominationError as _;
85 +     use bitcoin::amount::ParseDenominationError as _;
   |

error[E0432]: unresolved import `bitcoin::locktime::relative::IncompatibleHeightError`
   --> bitcoin/tests/check-re-exports.rs:186:9
    |
186 |     use bitcoin::locktime::relative::IncompatibleHeightError as _;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `IncompatibleHeightError` in `blockdata::locktime::relative`

error[E0432]: unresolved import `bitcoin::locktime::relative::IncompatibleTimeError`
   --> bitcoin/tests/check-re-exports.rs:187:9
    |
187 |     use bitcoin::locktime::relative::IncompatibleTimeError as _;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `IncompatibleTimeError` in `blockdata::locktime::relative`

error[E0432]: unresolved import `bitcoin::relative::IncompatibleHeightError`
   --> bitcoin/tests/check-re-exports.rs:236:9
    |
236 |     use bitcoin::relative::IncompatibleHeightError as _;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `IncompatibleHeightError` in `blockdata::locktime::relative`

error[E0432]: unresolved import `bitcoin::relative::IncompatibleTimeError`
   --> bitcoin/tests/check-re-exports.rs:237:9
    |
237 |     use bitcoin::relative::IncompatibleTimeError as _;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `IncompatibleTimeError` in `blockdata::locktime::relative`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `bitcoin` (test "check-re-exports") due to 12 previous errors
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
</details>

--- a/bitcoin/src/blockdata/mod.rs
+++ b/bitcoin/src/blockdata/mod.rs
@@ -84,8 +84,9 @@ pub mod locktime {
         pub use units::locktime::relative::{error, LockTime, NumberOf512Seconds, NumberOfBlocks};
         #[doc(no_inline)]
         pub use units::locktime::relative::{
-            DisabledLockTimeError, InvalidHeightError, InvalidTimeError, IsSatisfiedByError,
-            IsSatisfiedByHeightError, IsSatisfiedByTimeError, TimeOverflowError,
+            DisabledLockTimeError, IncompatibleHeightError, IncompatibleTimeError,
+            InvalidHeightError, InvalidTimeError, IsSatisfiedByError, IsSatisfiedByHeightError,
+            IsSatisfiedByTimeError, TimeOverflowError,
         };
 
         #[deprecated(since = "TBD", note = "use `NumberOfBlocks` instead")]

--- a/bitcoin/src/lib.rs
+++ b/bitcoin/src/lib.rs
@@ -241,9 +241,14 @@ pub mod amount {
     #[doc(inline)]
     pub use units::amount::{Amount, AmountDecoder, AmountEncoder, SignedAmount};
     #[doc(no_inline)]
-    pub use units::amount::{
-        AmountDecoderError, Denomination, Display, OutOfRangeError, ParseAmountError,
-        ParseDenominationError, ParseError,
+    pub use units::amount::{Denomination, Display};
+
+    #[doc(no_inline)]
+    pub use self::error::{
+        AmountDecoderError, BadPositionError, InputTooLargeError, InvalidCharacterError,
+        MissingDenominationError, MissingDigitsError, OutOfRangeError, ParseAmountError,
+        ParseDenominationError, ParseError, PossiblyConfusingDenominationError, TooPreciseError,
+        UnknownDenominationError,
     };
 
     /// Error types for bitcoin amounts.

